dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,237 @@
+/*
+ * PROJECT:     ReactOS Shell32
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Defining shell policy data
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+/* FIXME: Magic numbers */
+
+DEFINE_POLICY( REST_NORUN,                       "Explorer",       "NoRun"                            ),
+DEFINE_POLICY( REST_NOCLOSE,                     "Explorer",       "NoClose"                          ),
+DEFINE_POLICY( REST_NOSAVESET,                   "Explorer",       "NoSaveSettings"                   ),
+DEFINE_POLICY( REST_NOFILEMENU,                  "Explorer",       "NoFileMenu"                       ),
+DEFINE_POLICY( REST_NOSETFOLDERS,                "Explorer",       "NoSetFolders"                     ),
+DEFINE_POLICY( REST_NOSETTASKBAR,                "Explorer",       "NoSetTaskbar"                     ),
+DEFINE_POLICY( REST_NODESKTOP,                   "Explorer",       "NoDesktop"                        ),
+DEFINE_POLICY( REST_NOFIND,                      "Explorer",       "NoFind"                           ),
+DEFINE_POLICY( REST_NODRIVES,                    "Explorer",       "NoDrives"                         ),
+DEFINE_POLICY( REST_NODRIVEAUTORUN,              "Explorer",       "NoDriveAutoRun"                   ),
+DEFINE_POLICY( REST_NODRIVETYPEAUTORUN,          "Explorer",       "NoDriveTypeAutoRun"               ),
+DEFINE_POLICY( REST_NONETHOOD,                   "Explorer",       "NoNetHood"                        ),
+DEFINE_POLICY( REST_STARTBANNER,                 "Explorer",       "NoStartBanner"                    ),
+DEFINE_POLICY( REST_RESTRICTRUN,                 "Explorer",       "RestrictRun"                      ),
+DEFINE_POLICY( REST_NOPRINTERTABS,               "Explorer",       "NoPrinterTabs"                    ),
+DEFINE_POLICY( REST_NOPRINTERDELETE,             "Explorer",       "NoDeletePrinter"                  ),
+DEFINE_POLICY( REST_NOPRINTERADD,                "Explorer",       "NoAddPrinter"                     ),
+DEFINE_POLICY( REST_NOSTARTMENUSUBFOLDERS,       "Explorer",       "NoStartMenuSubFolders"            ),
+DEFINE_POLICY( REST_MYDOCSONNET,                 "Explorer",       "MyDocsOnNet"                      ),
+DEFINE_POLICY( REST_NOEXITTODOS,                 "WinOldApp",      "NoRealMode"                       ),
+DEFINE_POLICY( REST_ENFORCESHELLEXTSECURITY,     "Explorer",       "EnforceShellExtensionSecurity"    ),
+DEFINE_POLICY( REST_NOCOMMONGROUPS,              "Explorer",       "NoCommonGroups"                   ),
+DEFINE_POLICY( REST_LINKRESOLVEIGNORELINKINFO,   "Explorer",       "LinkResolveIgnoreLinkInfo"        ),
+DEFINE_POLICY( REST_NOWEB,                       "Explorer",       "NoWebMenu"                        ),
```

Comment:
The separate-process policy goes here:
```suggestion
DEFINE_POLICY( REST_SEPARATEDESKTOPPROCESS,      "Explorer",       "SeparateProcess"                  ),
DEFINE_POLICY( REST_NOWEB,                       "Explorer",       "NoWebMenu"                        ),
```